sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -1491,25 +1491,31 @@ public async Task ListBlobsFlatSegmentAsync_Deleted()
         {
             await using DisposingContainer test = await GetTestContainerAsync();
 
-            await EnableSoftDelete();
-            var blobName = GetNewBlobName();
-            AppendBlobClient blob = InstrumentClient(test.Container.GetAppendBlobClient(blobName));
-            await blob.CreateAsync();
-            await blob.DeleteAsync();
+            try
```

Comment:
Nit - should we create something that automatically enables/disables via IDisposable so folks always set this up with a `using` statement and can't forget to clean up when they're done?